### ISSUE TITLE
RDFNet training details

### ISSUE THREAD

**Issue by Linbooo** [SeongjinPark/RDFNet#6]
Hi Park,
   Regarding the training details of RDFNet for NYUD, I want to ask some questions as listed below:
     1. Which pretrained data did you use? ImageNet?
     2. How did you exactly preprocess the training data in terms of normalization? I only used ImageNet statistics provided from internet, and I did not see such step in your code.
    3. Which batch size and number of epochs/iterations did you use?
    4. You mentioned batch normalization in another Issue, did you implement it in RDFNet decoder which is not described in the paper? or you meant the batchnorm in ResNet backbone? In this case, did you freeze them all during training?
    5.  How did you implement Dropout? Did you have 9 dropouts in total? meaning that one dropout is placed before each MMFNet(there are 8 MMFNets in total) and last one placed before prediction? I have tried few strategies on my own for using dropout in RDFNet, but none of them showed improvement compared to no-dropout RDF.
   6. What are the random scale parameters during TRAINING? 0.6-1.2?

   A lot of questions are asked, hopefully you can recall these details :) I have spent a lot of time trying to reproduce your result but the results are not even close to yours. Really appreciate it if you can help me out with these questions, thanks.
